neutrons/hyspecppt
# Basegui

# Short description of the changes:

Added baseGUI and all the related tests in this PR for [EWM 8617](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20(Change%20Management)#action=com.ibm.team.workitem.viewWorkItem&id=8617)

Tests include checking label names and line editable for all widgets. Combo box is properly initialized.

# Long description of the changes:


# Check list for the pull request

# Check list for the reviewer

# Manual test for the reviewer

To test, git pull this branch. 
`cd hyspecppt`
`conda env create`
`pip install -e .`
type `hyspecppt` baseGUI should pop up
`pytest` all tests should pass.
# References
